### PR TITLE
getActionType returns wrong order of params

### PR DESCRIPTION
**Pull request by orm109**
https://github.com/stellarjs/stellarjs/issues/126

### TODO:
- [x] need to check tests

### CODE REVIEW

File: packages/client-redux/test/getActionType.test.js [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+                resource: 'resource',
+                method: 'get',
+                path: 'path',
+                payload,
+                options,
+            });
+        }
+        const x = getActionType(testFn);
+
+        expect(x).toBe('resource:path:get');
```
Comment (sloops77) [inline packages/client-redux/test/getActionType.test.js]: prefer `expect(getActionType(testFn)).toEqual('resource:path:get')`

everywhere

Comment (orm109) [reply packages/client-redux/test/getActionType.test.js]: ok
i usually use toBe for strings ... 
but i will change it

Comment (orm109) [reply packages/client-redux/test/getActionType.test.js]: done 👍

File: packages/client-redux/test/getActionType.test.js [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+    function testFn(payload, options, dispatch = () => null) {
+        return dispatch({
+            resource: 'resource',
+            method: 'get',
+            path: 'path',
+            payload,
+            options,
+          });
+      }
+    const x = getActionType(testFn);
```
Comment (sloops77) [inline packages/client-redux/test/getActionType.test.js]: get rid of the x

Comment (orm109) [reply packages/client-redux/test/getActionType.test.js]: sure
changed to a name which makes sense

**Pull request merged**